### PR TITLE
Test navigator.pdfViewerEnabled/plugins/mimeTypes

### PR DESCRIPTION
**Pull request by domenic**
Follows https://github.com/whatwg/html/pull/6738.

/cc @annevk @mfreed7 . Mason might want to take over these tests and move them into a Chromium CL since I just wrote them blind and there might be bugs.

### CODE REVIEW

**Review by annevk — APPROVED** [review]

**Comment by domenic** [comment]
This is subsumed into https://github.com/web-platform-tests/wpt/pull/29652 (Chromium export) so I'll close it, but I'll keep an eye on that Chromium CL to make sure it lands!

**Comment by mfreed7** [comment]
> This is subsumed into #29652 (Chromium export) so I'll close it, but I'll keep an eye on that Chromium CL to make sure it lands!

Thanks! So will I. I'm just waiting for the approvals on the I2S and I'll land it.

**Pull request closed without merging**
